modules/4337/test/erc4337/ERC4337ModuleExisting.spec.ts

```diff
@@ -9,20 +9,22 @@ describe('Safe4337Module - Existing Safe', () => {
   const setupTests = deployments.createFixture(async ({ deployments }) => {
     await deployments.fixture()
 
-    const [user1] = await ethers.getSigners()
-    const entryPoint = await getEntryPoint()
+    const [user1, relayer] = await ethers.getSigners()
+    let entryPoint = await getEntryPoint()
+    entryPoint = entryPoint.connect(relayer)
     const module = await getSafe4337Module()
     const safe = await getTestSafe(user1, await module.getAddress(), await module.getAddress())
 
     return {
       user1,
       safe,
+      relayer,
       validator: module,
       entryPoint,
     }
   })
 
-  describe('executeUserOp - existing account', () => {
+  describe('handleOps [- existing] account', () => {
```

Comment:
Looks like a search and replace change?
```suggestion
  describe('handleOps - existing account', () => {
```

Reply:
Don't know what happened here